src/SemanticLog/SemanticResourceInvokerAdapter.php

```
@@ -0,0 +1,82 @@
+
+        $openId = $this->semanticLogger->open($context);
+
+        try {
+            $result = $this->invoker->invoke($request);
+
+            $closeContext = new ResourceCompleteContext(
+                $resourceClass,
+                $resourceMethod,
+                $result->code,
+                /** @var array<string, mixed> */
+                (array) $result->body,
```

PHPStan (2.1.17)

Parameter #4 $body of class BEAR\Resource\SemanticLog\ResourceCompleteContext constructor expects array<string, mixed>, array<mixed, mixed> given.


(argument.type)